src/olympia/constants/categories.py

```diff
@@ -524,6 +525,8 @@ def _immutable(self, *args):
         },
     },
 }
+CATEGORIES[FIREFOX.id][_ADDON_PERSONA] = copy.copy(CATEGORIES[FIREFOX.id][ADDON_STATICTHEME])
+CATEGORIES[ANDROID.id][_ADDON_PERSONA] = copy.copy(CATEGORIES[ANDROID.id][ADDON_STATICTHEME])
```

Comment:
You shouldn't need the `ANDROID` copy - it wasn't there before - frontend just assumed the same for android as desktop.